scripts/format

```diff
@@ -5,5 +5,6 @@ export PREFIX="poetry run python -m "
 
 set -x
 
-${PREFIX}isort commitizen tests
+# ${PREFIX}isort commitizen tests
```

Comment:
Should we just remove it instead of commenting it out?

Reply:
yep